recipe/meta.yaml

```diff
@@ -30,24 +52,29 @@ build:
     - '*'
 
 outputs:
-  - name: sysroot_{{ target_platform }}
+  - name: sysroot_{{ cross_target_platform }}
     script: build-sysroot.sh
     build:
       noarch: generic
```

Comment:
Can you add a `track_features` entry here?

Reply:
Should the cdt version be in it?

Reply:
Yes